scripts/build/companion_libs/330-gettext.sh

```diff
@@ -118,6 +118,14 @@ do_gettext_backend() {
         --disable-csharp                                    \
         --without-emacs                                     \
         --disable-openmp                                    \
+        --with-included-libxml                              \
+        --with-included-gettext                             \
+        --with-included-glib                                \
+        --with-included-libcroco                            \
+        --with-included-libunistring                        \
+        --with-libncurses-prefix="${prefix}"                \
```

Comment:
Very interesting. Gettext is shipped with fixed (w.r.t. #443) version of gnulib, i.e. it properly selects static library libncurses.a located in ${prefix}, ignoring shiny tasty shared library /usr/lib/libncurses.so.